visualization/app/codeCharta/ui/codeMap/rendering/geometryGenerator.ts

```diff
@@ -176,11 +188,13 @@ export class GeometryGenerator {
             uvs[i * uvDimension + 0] = data.uvs[i].x;
             uvs[i * uvDimension + 1] = data.uvs[i].y;
 
-            let color: THREE.Vector3 = RenderingUtil.colorToVec3(data.colors[i]);
+            if (data.colors[i]) {
```

Comment:
@mrpatpat Do you know if the `if` statement could cause any problems? Just in the tests, there was given a `colors`-array with 12 hex color-values and 12 undefined/null values... Could you please check that as well? Just remove the `if`-statement and you'll see which test fails..

Reply:
Solved